gtsam/linear/tests/testAcceleratedPowerMethod.cpp

```diff
@@ -0,0 +1,110 @@
+/* ----------------------------------------------------------------------------
+
+ * GTSAM Copyright 2010-2019, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+ * See LICENSE for the license information
+
+ * -------------------------------------------------------------------------- */
+
+/**
+ * testPowerMethod.cpp
+ *
+ * @file   testAcceleratedPowerMethod.cpp
+ * @date   Sept 2020
+ * @author Jing Wu
+ * @brief  Check eigenvalue and eigenvector computed by accelerated power method
+ */
+
+#include <gtsam/base/Matrix.h>
+#include <gtsam/base/VectorSpace.h>
+#include <gtsam/inference/Symbol.h>
+#include <gtsam/linear/GaussianFactorGraph.h>
+#include <gtsam/linear/AcceleratedPowerMethod.h>
+
+#include <CppUnitLite/TestHarness.h>
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+#include <Eigen/Eigenvalues>
+
+#include <iostream>
+#include <random>
+
+using namespace std;
+using namespace gtsam;
+using symbol_shorthand::X;
+
+/* ************************************************************************* */
+TEST(AcceleratedPowerMethod, acceleratedPowerIteration) {
+  // test power iteration, beta is set to 0
+  Sparse A(6, 6);
+  A.coeffRef(0, 0) = 6;
+  A.coeffRef(0, 0) = 5;
+  A.coeffRef(0, 0) = 4;
+  A.coeffRef(0, 0) = 3;
+  A.coeffRef(0, 0) = 2;
+  A.coeffRef(0, 0) = 1;
+  Vector initial = Vector6::Zero();
+  const Vector6 x1 = (Vector(6) << 1.0, 0.0, 0.0, 0.0, 0.0, 0.0).finished();
```

Comment:
Fixed

Reply:
It might be better here to make ```initial``` a hard-coded vector instead of a random sample -- this would make it easier to reproduce failing tests at different times, different machines, etc.